### ISSUE TITLE
polkadot-v0.9.2 support

### ISSUE THREAD

**Issue by GopherJ** [paritytech/subxt#282]
hi, we upgraded to polkadot-v0.9.2 and it seems substrate-subxt is then broken:

```
Could not decode `RuntimeMetadata`, variant doesn\'t exist" 
```

**Comment by ascjones** [comment]
Ah yes that's because a new metadata version has been added, so the metadata dependency needs updating

**Comment by shekohex** [comment]
Any updates on this? we hit the same issue with our current node!

**Comment by ascjones** [comment]
Since there is no new release of `frame-metadata` on crates.io, we will need to use github dependencies in the meantime. I'll have a look today.

**Comment by ascjones** [comment]
I've started on this https://github.com/paritytech/substrate-subxt/pull/286. Needs a bit more work to get the tests passing.

**Comment by dkuanyshbaev** [comment]
Hi! We have the same problem. Any updates? Thank you.

**Comment by ascjones** [comment]
I'm very busy with my substrate PR: https://github.com/paritytech/substrate/pull/8615 which will introduce type information and solve this problem once and for all. Once I have that ready for review I will look at this again.

**Comment by GopherJ** [comment]
now we are on polkadot-v0.9.8 and subxt is still using very old dependencies, any timeline for this? we started to use polkadotjs for new offchain stuff because subxt looks unmaintained for a while and we are worried about it

**Comment by ascjones** [comment]
No need to be worried, it is still waiting on https://github.com/paritytech/substrate/pull/8615/files, which should be merged in the next few weeks

**Comment by conectado** [comment]
> No need to be worried, it is still waiting on https://github.com/paritytech/substrate/pull/8615/files, which should be merged in the next few weeks

Hi! I hope I'm not bothering, is there any news on this since the PR was merged?

**Comment by ascjones** [comment]
It's been merged, I'm just working on https://github.com/paritytech/substrate-subxt/pull/294 which should fix this.

**Comment by ascjones** [comment]
Closed by #294
